ui/src/main/resources/edu/wpi/grip/ui/templates/cpp/operations/CV_GaussianBlur.vm

```diff
@@ -0,0 +1,16 @@
+	/**
+	 * Performs a Gaussian blur on the image.
+	 * @param src the image to blur.
+	 * @param kSize the kernel size.
+	 * @param sigmaX the deviation in X for the Gaussian blur.
+	 * @param sigmaY the deviation in Y for the Gaussian blur.
+	 * @param borderType pixel extrapolation method.
+	 * @param dst the output image.
+	 */
+	void $tMeth.name($step.name())(Mat *src, Size *kSize, double *sigmaX, double *sigmaY, int
+	*borderType, Mat *dst){
+		if(!kSize){
+			Size kSize(1,1);
```

Comment:
Is this valid? Seems like you're creating a `Size` object in this scope but doing nothing with it.